sm-core/src/main/java/com/salesmanager/core/business/modules/cms/content/ContentAssetsManager.java

```diff
@@ -13,8 +13,8 @@
 public interface ContentAssetsManager extends AssetsManager, FileGet, FilePut, FileRemove, FolderPut, FolderList, FolderRemove, Serializable {
     public static final char UNIX_SEPARATOR = '/';
     public static final char WINDOWS_SEPARATOR = '\\';
-    public static String DEFAULT_BUCKET_NAME = "shopizer";
-    public static String DEFAULT_REGION_NAME = "us-east-1";
+    public static String DEFAULT_BUCKET_NAME = "topbuys";
+    public static String DEFAULT_REGION_NAME = "fra1";
```

Comment:
what's fra1?

Reply:
I think this is Frankfurt, Germany 
https://www.digitalocean.com/docs/platform/availability-matrix/

Reply:
I am not sure,  could it be better to pass this configuration from env variable or db?just in case we need to change it without code deployment, what do you think?

Reply:
it's the region name for digital ocean it's for Frankfurt

Reply:
It's already passed in app props file, but these values was set as default from old configuration in case the props not configured it will be used, so I let it and changed it to be our values